notebooks/ingestion/Loading.ipynb

```diff
@@ -0,0 +1,275 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Loading the data into the DB.\n",
+    "\n",
+    "Following the concepts on the different [means](/HowToGuides/start/Ingestion) of ingesting the data,\n",
+    "we will use the Cookbook dataset to have a working example with each of the type defined.\n",
+    "\n",
+    "Starting from 2 input files, we will take the data into the ApertureDB using the 3 methods mentioned in the link above.\n",
+    "- Ingestion with DataModel.\n",
```

Comment:
Do we explain the concept of data model anywhere in our docs?

Reply:
Similarly, what is a query generator -> do we have in docs somewhere? It doesn't need to go here but perhaps - we need to add a page in this section about this and data models before starting examples?

Reply:
yup, we have some explanation of the 3 concepts in here: https://docs.aperturedata.io/HowToGuides/start/Ingestion